### DISCUSSION TITLE
grafana-cli panel update commands

### DISCUSSION THREAD

**Discussion by nicolastakashi** [grafana/grafana#47053] [Dashboards, Panels & Visualizations]
It would be really awesome if there are two commands to handle panel updates on grafana-cli

## check update panel

```bash
grafana-cli panel upgrade-check --dashboard-path ./dashboards
```

## update panel
```bash
grafana-cli panel upgrade --dashboard-path ./dashboards
```

This will enable teams to handle panels update in an automated way, instead of needing to go to the panel settings and click on the update panel.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
